CMakeLists.txt

```diff
@@ -163,8 +163,8 @@ else (APPLE)
       set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -mno-ms-bitfields -O2 -DNDEBUG -DQT_NO_DEBUG")
       set(CMAKE_EXE_LINKER_FLAGS "-Wl,--large-address-aware")
    else (MINGW)
-      set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -fPIC -g")
-      set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -fPIC -O2 -DNDEBUG -DQT_NO_DEBUG")
+      set(CMAKE_CXX_FLAGS_DEBUG   "-std=gnu++11 -fno-inline -fPIC -g")
+      set(CMAKE_CXX_FLAGS_RELEASE "-std=gnu++11 -fno-inline -fPIC -O2 -DNDEBUG -DQT_NO_DEBUG")
```

Comment:
The reason was that I see a lot of these warnings:
```
all.h.gch: not used because `__NO_INLINE__' not defined [-Winvalid-pch]
```
Can this be fixed in **all.h.gch** maybe?